### PR TITLE
style: rebuilt homepage and added sliders with new cards.

### PR DESCRIPTION
**Pull request by tushartiwari7**
Work Done: 
- Completely changed the UI of the landing Page including the Color Palette.
- Added a short video in Banner.
- A Mosaic of categories with links to Product page with selected Category.
- A slider of Best Selling Products.
- Company ethics and some information about furniture store.

### CODE REVIEW

File: src/components/Banner/Banner.jsx [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
+      </div>
+    </section>
+  );
+};
+
+/**
+ * 
     <Link to='/products'>
       <img src="https://godrejinterio.com/imagestore/B2C/EspotImages/Images/Banners/Q4_HomePage-01.webp" alt="Sofa_Sets"
       width={'100%'}
```
Comment (ashishnnnnn) [inline src/components/Banner/Banner.jsx]: This Commented code can be omitted..

File: src/components/Categories/Categories.jsx [hunk 2]
```diff
@@ -5,18 +5,23 @@ import { useProductCategory } from "../../context";
...
-      </ul>
-    </div>
+    <ul className="list category-list fs-s grid full-width pos-rel">
+      {categories.map(({ _id, categoryName, description, img }, idx) => (
+        <Link
+          to={`/products?categoryName=${categoryName}`}
+          key={_id}
+          className={`category pos-rel ${idx === 3 ? "category-special" : ""}`}
+        >
+          <img src={img} />
```
Comment (ashishnnnnn) [inline src/components/Categories/Categories.jsx]: Image can have alt  attribute..

**Review by ashishnnnnn — APPROVED** [review]
Good Work..
Liked your work on UI part.. :+1:

File: src/components/Banner/Banner.jsx (hunk 1 above)
Comment (tushartiwari7) [reply src/components/Banner/Banner.jsx]: Done

File: src/components/Categories/Categories.jsx (hunk 2 above)
Comment (tushartiwari7) [reply src/components/Categories/Categories.jsx]: Done

**Review by JuvitaSaini09 — APPROVED** [review]
code is very clean,looks good to merge

**Review by Bhakti199 — APPROVED** [review]
- This code looks fine to me.
- Proper color palette is used.
- Well written and structured code.
- UI looks cool.
- This PR is good to merge.

**Pull request merged**